hello-world-ux-aspects/README.md

```diff
@@ -0,0 +1,50 @@
+# HelloWorldUxAspects
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 1.5.3.  
+
+This application utilizes the UX Aspects user interface framework and provides a great place to start for building modern, 
+responsive, mobile big data applications on the web.
+
+# Documentation and Examples
+
+See the documentation site for full details and interactive examples of UX Aspects components:
+
+[https://pages.github.houston.softwaregrp.net/caf/ux-aspects-micro-focus/#/landing](https://pages.github.houston.softwaregrp.net/caf/ux-aspects-micro-focus/#/landing)
+
+## Prerequisites
+
+* [Node.js](https://nodejs.org/) - this provides the `npm` command, which is used to install application dependencies.
+
+* [Git](https://git-scm.com/) - if building from source, Git is the best way to download the code.
+
+## Installation
+
+1. Clone the [CX](https://github.com/MicroFocus/CX) repository.
+
+2. Switch to the `hello-world-ux-aspects` directory.
+
+3. Configure the @micro-focus scope in NPM:
```

Comment:
Shouldn't need to do this step. The npmrc file does this for us. Just leave the note in about contactin us if it is not accessible